lib/datadog/version.rb

```diff
@@ -1,13 +1,12 @@
 # frozen_string_literal: true
 
-module DDTrace
+module Datadog
   module VERSION
     MAJOR = 1
     MINOR = 20
     PATCH = 0
     PRE = nil
     BUILD = nil
-    # PRE and BUILD above are modified for dev gems during gem build GHA workflow
```

Comment:
Minor: I think this comment is helpful to keep around, as otherwise it doesn't look like `PRE` or `BUILD` are very useful.